package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rtpmidi",
-  "version": "0.1.9",
+  "version": "0.1.8",
```

Comment:
Could you please raise the version number? Maybe it makes sense to just start with 1.0.0 with these changes and continue with proper semantic versioning going forward.

Reply:
I would say v1.0.0 would need to implement most of the spec and have testing, but sure will increment the version!

Reply:
Hey, I'd prefer 1.0.0 because the changes are quite extensive and I wouldn't want people who use the 0.. version to be updated automatically because of the semantic versioning.

Reply:
Yeah sure!